### PR TITLE
consistent code and tutorial pages

### PR DESCRIPTION
**Pull request by 7opf**
* Closes https://github.com/ZebraDevs/Zebra-Iota-Edge-SDK/issues/98
* Closes https://github.com/ZebraDevs/Zebra-Iota-Edge-SDK/issues/99

### CODE REVIEW

File: identity-enabler/deviceId-mobile-app/src/pages/CreatePresentation.svelte [hunk 1]
```diff
@@ -37,32 +33,19 @@
...
         navigate("/tutorial");
     }
 
     function showJSON() {
-        modalStatus.set({
-            active: true,
-            type: "code",
-            props: { code: JSON.stringify(vp, null, 2) }
-        });
+        navigate("/code", { state: { code: JSON.stringify(vp, null, 2) } });
```
Comment (jmcanterafonseca-iota) [inline identity-enabler/deviceId-mobile-app/src/pages/CreatePresentation.svelte:41]: I don't know if navigating to a different page is a good idea. Removing the modal now leads to a very bad performance when going back to the data matrix page ....

File: identity-enabler/deviceId-mobile-app/src/pages/Tutorial.svelte [hunk 2]
```diff
@@ -6,45 +6,45 @@
...
+    let title = "";
 
     onMount(async () => {
         loadingScreen.set("Loading...");
         try {
-            code = await getMarkdownContent(TUTORIAL_URL);
+            const tutorialMd: string = await getMarkdownContent(TUTORIAL_URL);
+            const endOfTitleIdx = tutorialMd.indexOf("\n");
+            title = tutorialMd.substring(2, endOfTitleIdx);
+            markdown = tutorialMd.substring(endOfTitleIdx + 1);
```
Comment (jmcanterafonseca-iota) [inline identity-enabler/deviceId-mobile-app/src/pages/Tutorial.svelte:18]: is this really necessary, to depend on the structure of the input tutorial? can we just have a title and avoid potential code breaking if the tutorial structure changes?

File: identity-enabler/holder-mobile-app/src/pages/Tutorial.svelte [hunk 3]
```diff
@@ -6,52 +6,65 @@
...
+    let title = "";
 
     onMount(async () => {
         loadingScreen.set("Loading...");
         try {
-            code = await getMarkdownContent(TUTORIAL_URL);
+            const tutorialMd: string = await getMarkdownContent(TUTORIAL_URL);
+            const endOfTitleIdx = tutorialMd.indexOf("\n");
+            title = tutorialMd.substring(2, endOfTitleIdx);
+            markdown = tutorialMd.substring(endOfTitleIdx + 1);
```
Comment (jmcanterafonseca-iota) [inline identity-enabler/holder-mobile-app/src/pages/Tutorial.svelte:18]: same coment

File: identity-enabler/verifier-mobile-app/src/pages/Tutorial.svelte [hunk 4]
```diff
@@ -6,52 +6,65 @@
...
+    let title = "";
 
     onMount(async () => {
         loadingScreen.set("Loading...");
         try {
-            code = await getMarkdownContent(TUTORIAL_URL);
+            const tutorialMd: string = await getMarkdownContent(TUTORIAL_URL);
+            const endOfTitleIdx = tutorialMd.indexOf("\n");
+            title = tutorialMd.substring(2, endOfTitleIdx);
+            markdown = tutorialMd.substring(endOfTitleIdx + 1);
```
Comment (jmcanterafonseca-iota) [inline identity-enabler/verifier-mobile-app/src/pages/Tutorial.svelte:18]: same comment

File: identity-enabler/holder-mobile-app/src/pages/CreatePresentation.svelte [hunk 5]
```diff
@@ -47,32 +44,19 @@
...
     }
 
     function showJSON() {
-        modalStatus.set({
-            active: true,
-            type: "code",
-            props: { code: presentationJSON }
-        });
+        navigate("/code", { state: { code: presentationJSON } });
     }
```
Comment (jmcanterafonseca-iota) [inline identity-enabler/holder-mobile-app/src/pages/CreatePresentation.svelte:53]: same comment

**Review by jmcanterafonseca-iota — CHANGES_REQUESTED** [review]
there have been performance regressions in this PR

File: identity-enabler/deviceId-mobile-app/src/pages/Tutorial.svelte (hunk 2 above)
Comment (7opf) [reply identity-enabler/deviceId-mobile-app/src/pages/Tutorial.svelte:18]: Yes, but then there are two titles, one in the header and one in the rendered markdown.

File: identity-enabler/holder-mobile-app/src/pages/Tutorial.svelte (hunk 3 above)
Comment (7opf) [reply identity-enabler/holder-mobile-app/src/pages/Tutorial.svelte:18]: See https://github.com/ZebraDevs/Zebra-Iota-Edge-SDK/pull/101#discussion_r793371211

File: identity-enabler/verifier-mobile-app/src/pages/Tutorial.svelte (hunk 4 above)
Comment (7opf) [reply identity-enabler/verifier-mobile-app/src/pages/Tutorial.svelte:18]: See https://github.com/ZebraDevs/Zebra-Iota-Edge-SDK/pull/101#discussion_r793371211

File: identity-enabler/holder-mobile-app/src/pages/CreatePresentation.svelte (hunk 5 above)
Comment (7opf) [reply identity-enabler/holder-mobile-app/src/pages/CreatePresentation.svelte:53]: Performance issues will get addressed in future PR https://github.com/ZebraDevs/Zebra-Iota-Edge-SDK/pull/76

File: identity-enabler/deviceId-mobile-app/src/pages/CreatePresentation.svelte (hunk 1 above)
Comment (7opf) [reply identity-enabler/deviceId-mobile-app/src/pages/CreatePresentation.svelte:41]: Performance issues will get addressed in future PR https://github.com/ZebraDevs/Zebra-Iota-Edge-SDK/pull/76

**Pull request merged**